docs/model_inputs_outputs.md

```diff
@@ -0,0 +1,11 @@
+# Model Inputs and Outputs, Shapes and Layouts {#openvino_docs_model_inputs_outputs}
+
+Users interact with a model by passing data to its _inputs_ and retrieving data from its _outputs_ after the inference. A model may have one or multiple inputs and outputs. Normally, in OpenVINO™ toolkit, all inputs and outputs in the converted model are identified in the same way as in the original framework model. 
+
+OpenVINO uses the _names_ of tensors for identification. Depending on the framework, the names of tensors are formed differently. Learn how to retrieve specific input/output depending on the framework here.
```

Comment:
Where is `here`? We are not explaining it here.

Reply:
@mvafin is working on the description, will add when it's done